### DISCUSSION TITLE
A2DP microphone

### DISCUSSION THREAD

**Discussion by vorosj** [pschatzmann/ESP32-A2DP#545] [Q&A] [answered]
How can I send audio stream to a PC, for example generated sine -> A2DP -> PC? The base-generator-a2dp example works using a bluetooth speaker, but how to connect the esp32 to a PC? I mean the PC recognizes it as a Bluetooth microphone?

**Comment by pschatzmann** [comment, marked as answer]
I am sorry, but I can't provide any PC support: [Google is your friend](https://www.google.com/search?q=A2DP+Bluetooth+Sink+Windows&sca_esv=4250fb469ad3ef44&sca_upv=1&sxsrf=ACQVn08l8Qj6Nd1A3peQNc_PGeWujMP4kw%3A1713675460025&ei=xJwkZruSAeeD9u8PuZquqA4&ved=0ahUKEwi7hLjIwtKFAxXngf0HHTmNC-UQ4dUDCBA&uact=5&oq=A2DP+Bluetooth+Sink+Windows&gs_lp=Egxnd3Mtd2l6LXNlcnAiG0EyRFAgQmx1ZXRvb3RoIFNpbmsgV2luZG93czIEECMYJzIGEAAYFhgeMgYQABgWGB4yBhAAGBYYHjIGEAAYFhgeMgsQABiABBiGAxiKBTIIEAAYogQYiQUyCBAAGIAEGKIEMggQABiABBiiBEjsrwNQ6iRYikFwBHgBkAEAmAGGAaABzAWqAQM1LjK4AQPIAQD4AQH4AQKYAgugArkGwgIKEAAYsAMY1gQYR8ICChAhGKABGMMEGArCAgYQABgIGB6YAwCIBgGQBgiSBwM0LjegB8oq&sclient=gws-wiz-serp) ...

**Comment by vorosj** [comment]
Thank you. 
I assumed wrongly that Win11 has a built-in function, so I just need a different parameter in the ESP32 code and a wireless Bluetootn mic is easy job. :-) 
Strange, that it's not the case.
